rpc/server_auth.go

```diff
@@ -272,29 +293,32 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (interface{}, error) {
 	// usess pointers to time.Time causing parsing issues. For now we can just reparse the json jwt token into the claim.
 	_, _, err = jwtParser.ParseUnverified(outToken.Raw, claims)
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "error decoding claims: %s", err)
+		return "", nil, status.Errorf(codes.InvalidArgument, "error decoding claims: %s", err)
 	}
 
 	// We MUST validate claims here. We disabled claims validation in the parser above.
 	err = claims.Valid()
 	if err != nil {
-		return nil, status.Errorf(codes.Unauthenticated, "unauthenticated: %s", err)
+		return "", nil, status.Errorf(codes.Unauthenticated, "unauthenticated: %s", err)
 	}
 
 	entity, err := claims.Entity()
 	if err != nil {
-		return nil, err
+		return "", nil, err
 	}
 
-	// Pass the raw claims to the Context.
-	ctx = contextWithAuthClaims(ctx, claims)
-
-	// Pass the auth metadata to the context.
-	if claims.GetAuthMetadata() != nil {
-		ctx = contextWithAuthMetadata(ctx, claims.GetAuthMetadata())
+	// TODO(RSDK-886): start requiring this; until then, log it for observation.
```

Comment:
// TODO(RSDK-886): start requiring this; until then, log it for observation.

You're returning an error here?

Reply:
yeah removing. old